markdown.dtx

```diff
@@ -14929,6 +14933,7 @@ Example themes provided with the Markdown package include:
 %    ```` tex
 %    \documentclass{article}
 %    \usepackage[theme=witiko/dot]{markdown}
+%    TODO replace with l3keys functions
```

Comment:
`\setkeys{Gin}` configures the `graphicx` package. The graphicx package uses and will use `keyval`.

Reply:
```suggestion
```